### PR TITLE
Style fixes and final changes

### PR DESCRIPTION
**Pull request by adbugger**
Ideally this should have been pull #3 reopened. However, I'm not fully comfortable with reverting commits yet; anyway this is more to check for PEP8 compliance and whether Travis-CI works.

### CODE REVIEW

**Comment by adbugger** [comment]
Travis-CI works. Canceled builds due to incoming commits.

File: scipy/spatial/transform/rotation.py [hunk 1]
```diff
@@ -43,7 +45,8 @@ def __init__(self, quat, normalized=False):
             # which (0,0,0,1) in (x,y,z,w) format
             zero_norms = norms == 0
             if zero_norms.any():
-                warnings.warn("Found zero norm quaternions in input, replacing with identity quaternions.")
+                warnings.warn("""Found zero norm quaternions in input,
+                        replacing with identity quaternions.""")
                 self._quat[zero_norms] = np.array([0, 0, 0, 1])
```
Comment (nmayorov) [inline scipy/spatial/transform/rotation.py]: I would suggest to simply break a standard `"..."` string, at least this is how I usually saw it's done.

Also check https://www.python.org/dev/peps/pep-0008/#indentation for formatting guidelines.

**Comment by nmayorov** [comment]
How did you enable `CircleCI` builds?

**Comment by adbugger** [comment]
I didn't enable `CircleCI` yet, this is only `Travis-CI` as of now. I simply went to [https://travis-ci.org](https://travis-ci.org) and clicked on the plus button next to my repositories, and switched on the travis-ci service next to scipy in the list. I already had an account there, they have an option of making an account with github in any case.

For the configurations and tests, it is automatically using the `travis.yml` file that is already present in my fork.

I suspect there will be something similar for `Circle-CI` also, I'll look into it.

**Comment by larsoner** [comment]
Yes the procedure for Circle should be pretty much the same. The upstream
yaml files should work just fine once you enable Circle on your fork on
their website.

>

File: scipy/spatial/transform/rotation.py (hunk 1 above)
Comment (adbugger) [reply scipy/spatial/transform/rotation.py]: Breaking a standard `"..."` string with either `\` or using a multi-line string with triple quotes such as 
`'''abc`
`def'''` 
prints newlines and blank spaces also. This method prints the string on a single line.

Do correct me if there's some other method I'm missing.

Comment (nmayorov) [reply scipy/spatial/transform/rotation.py]: Well, in this case you should write it as:

```
warnings.warn("Found zero norm quaternions in input, 
               replacing with identity quaternions.")
```

Check that it works as expected.

Comment (adbugger) [reply scipy/spatial/transform/rotation.py]: That doesn't seem to work:
`SyntaxError: EOL while scanning string literal`

Comment (nmayorov) [reply scipy/spatial/transform/rotation.py]: Oh, I'm sorry. What I actually meant:
```Python
warnings.warn("Found zero norm quaternions in input, "
              "replacing with identity quaternions.")
```

Comment (adbugger) [reply scipy/spatial/transform/rotation.py]: Ah, that works. I'd only seen triple quoted string broken this way.

**Comment by adbugger** [comment]
At the very least it appears that `CircleCI` and `Travis CI` are all set up.

File: scipy/spatial/transform/rotation.py [hunk 2]
```diff
@@ -4,14 +4,16 @@
...
 
-    This class will include initializers from different representations, converters
-    and some useful algorithms such as quaternion slerp and rotation estimation.
+    This class will include initializers from different representations,
+    converters and some useful algorithms such as quaternion slerp and
+    rotation estimation.
 
-    For initializing Rotations usage of `from_...` methods such as `from_quaternion`
-    is recommended instead of using `__init__`.
+    For initializing Rotations usage of `from_...` methods such as
```
Comment (nmayorov) [inline scipy/spatial/transform/rotation.py:15]: I think it is not very precisely put, basically `__init__` initializes from quaternions only.

I would write simply something like: ``"To create an instance use one of `Rotation.from_...` class(or factory?) methods."`` My point is that "recommended" is not a right word here. I think my short phrase explains everything is there to know.

**Comment by nmayorov** [comment]
Let's wait for the checks to complete and merge. Don't bother with my comment about the docstring in here, but keep it in my mind.

**Pull request merged**